content/id/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -79,13 +81,33 @@ <h2>Kubernetes Basics Modules</h2>
           <div class="row">
             <div class="col-md-4">
               <div class="thumbnail">
-                <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
+                <a href="/id/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/id/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. Mengekspos aplikasimu secara publik</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_05.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. Scale up your app</h5></a>
```

Comment:
Missed translation?
```suggestion
                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. Menaikkan skala aplikasi anda</h5></a>
```

Reply:
Thanks, that is a nice translation.

Reply:
Wait, it should be scale-out, not scale-up. Because it is horizontally scaled, not vertically.

Reply:
penyekalaan is indonesian word https://kbbi.kemdikbud.go.id/entri/penyekalaan

you can make use of this word

Reply:
We may need to think of the variation scale up/down and scale out/in.

Reply:
We can use “penyekalaan naik/turun” or “perbesar/perkecil skala”

Reply:
```suggestion
                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. Penyekalaan naik aplikasimu</h5></a>
```